display-svga-x11.cpp:315:41: error: invalid conversion from 'int (*)(RTTHREAD, void*) {aka int (*)(RTTHREADINT*, void*)}'
to 'PFNRTTHREAD {aka int (__attribute__((__regparm__(0), __cdecl__)) *)(RTTHREADINT*, void*)}' [-fpermissive]                                                                                                                                           "X11 events");                                                                     
--- VirtualBox-6.1.6/src/VBox/Additions/x11/VBoxClient/display-svga-x11.cpp	2020-04-09 20:34:49.000000000 +0000
+++ /export/home/alp/srcs/oi-userland/components/sysutils/virtualbox/build/i86/src/VBox/Additions/x11/VBoxClient/display-svga-x11.cpp	2020-04-22 08:05:06.291862654 +0000
@@ -293,7 +293,7 @@
     }
 }
 
-int x11MonitorThreadFunction(RTTHREAD hThreadSelf, void *pvUser)
+DECLCALLBACK(int) x11MonitorThreadFunction(RTTHREAD hThreadSelf, void *pvUser)
 {
     (void)hThreadSelf;
     (void*)pvUser;
